index.bs

```diff
@@ -6799,7 +6802,7 @@ which form (or forms) it is in:
             <dfn id="dfn-xattr-argument-list" for="extended attribute" export>takes an argument list</dfn>
         </td>
         <td>
-            <code>[Constructor(double x, double y)]</code>
+            Not currently used; previously used by <code>[Constructor(double x, double y)]</code>
```

Comment:
Not even by NamedConstructor?

Reply:
Nope -- it uses named argument list, rather than just argument list:

> [The [NamedConstructor] extended attribute must either take an identifier or take a named argument list.](https://heycam.github.io/webidl/#NamedConstructor)

It's possible that implementer-specific attributes still use this (I haven't checked Blink's list); that's the only reason I didn't try to remove it entirely.

Reply:
D'oh!

Reply:
Interesting. We should probably create a thread to discuss removing this entirely then, and let implementers chime in there.

Reply:
The only other usage I could find is in the Blink-specific 'CustomConstructor' attribute, but I suspect we could probably replace usages of

```
[ CustomConstructor(float x, int y) ] interface Z { ... };
```

with

```
interface Z {
  [Custom] constructor(float x, int y);
  ...
};
```

and be rid of that attribute.